diff --git a/native/src/CMakeLists.txt b/native/src/CMakeLists.txt
index b5628f9..f8283ac 100644
--- a/native/src/CMakeLists.txt
+++ b/native/src/CMakeLists.txt
@@ -82,7 +82,9 @@
 endif()
 
 # Required files and directories
-include(GNUInstallDirs)
+if (NOT CMAKE_INSTALL_LIBDIR OR NOT CMAKE_INSTALL_INCLUDEDIR)
+    include(GNUInstallDirs)
+endif()
 
 # Source Tree
 set(SEAL_INCLUDES_DIR ${CMAKE_CURRENT_LIST_DIR}/native/src)
 
